SWLOR.Game.Server/Feature/SpawnDefinition/SpaceSpawnDefinition.cs

```diff
@@ -104,27 +104,32 @@ private void KorribanOrbit()
         {
             _builder.Create("SPACE_KORRIBAN_ORBIT", "Space - Korriban Orbit")
                 .AddSpawn(ObjectType.Creature, "pirate_night")
-                .WithFrequency(40)
+                .WithFrequency(20)
                 .RandomlyWalks()
                 .ReturnsHome()
 
                 .AddSpawn(ObjectType.Creature, "pirate_ferron")
-                .WithFrequency(5)
+                .WithFrequency(20)
                 .RandomlyWalks()
                 .ReturnsHome()
 
                 .AddSpawn(ObjectType.Creature, "pirate_storm")
-                .WithFrequency(1)
+                .WithFrequency(20)
                 .RandomlyWalks()
                 .ReturnsHome();
+
```

Comment:
Compilation error here. Need to kill the semi-colon on line 119. Would strongly recommend testing this locally by the way ;)  Reach out if you need help setting up a server - instructions are in the download package on the website. Can walk you through if needed.